examples/tutorial/jupyter/execution/pandas_on_dask/cluster/exercise_5.ipynb

```diff
@@ -0,0 +1,372 @@
+    "\n",
+    "In the next cell you can see how the EC2Cluster is being created. <b>Set your ``key_name`` and modify AWS settings as required before running it.</b>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask_cloudprovider.aws import EC2Cluster\n",
+    "\n",
+    "n_workers = 6\n",
+    "cluster = EC2Cluster(\n",
+    "    # AWS parameters\n",
+    "    key_name = \"\", # set your keyname\n",
+    "    region = \"us-west-2\",\n",
+    "    availability_zone = [\"us-west-2a\"],\n",
+    "    ami = \"ami-0387d929287ab193e\",\n",
+    "    instance_type = \"m5.24xlarge\",\n",
+    "    vpc = \"vpc-002bd14c63f227832\",\n",
+    "    subnet_id = \"subnet-09860dafd79720938\",\n",
+    "    filesystem_size = 200, # in GB\n",
+    "\n",
+    "    # DASK parameters\n",
+    "    n_workers = n_workers,\n",
+    "    docker_image = \"daskdev/dask:latest\",\n",
+    "    debug = True,\n",
+    "    security=False,\n",
+    ")\n",
+    "\n",
+    "scheduler_adress = cluster.scheduler_address\n",
+    "print(f\"Scheduler IP address of Dask cluster: {scheduler_adress}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After creating the cluster you need to connect to it. To do this you should put the ``EC2Cluster`` instance or the scheduler IP address in ``distributed.Client``.\n",
+    "\n",
+    "When you connect to the cluster, the workers may not be initialized yet, so you need to wait for them using ``client.wait_for_workers``.\n",
+    "\n",
+    "Then you can call ``client.ncores()`` and check which workers are available and how many threads are used for each of them."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from distributed import Client\n",
+    "\n",
+    "client = Client(cluster)\n",
+    "# Or use an IP address connection if the cluster instance is unavailable:\n",
+    "# client = Client(f\"{scheduler_adress}:8687\")\n",
+    "\n",
+    "client.wait_for_workers(n_workers)\n",
+    "client.ncores()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After successful initialization of the cluster, you need to configure it.\n",
+    "\n",
+    "You can use plugins to install any requirements into workers:\n",
+    "* [InstallPlugin](\"https://distributed.dask.org/en/stable/plugins.html#distributed.diagnostics.plugin.InstallPlugin\")\n",
+    "* [PipInstall](\"https://distributed.dask.org/en/stable/plugins.html#distributed.diagnostics.plugin.PipInstall\")\n",
+    "* [CondaInstall](\"https://distributed.dask.org/en/stable/plugins.html#distributed.diagnostics.plugin.CondaInstall\").\n",
```

Comment:
These links don't work for me. Maybe it can fix them.
```suggestion
    "* [InstallPlugin](https://distributed.dask.org/en/stable/plugins.html#distributed.diagnostics.plugin.InstallPlugin)\n",
    "* [PipInstall](https://distributed.dask.org/en/stable/plugins.html#distributed.diagnostics.plugin.PipInstall)\n",
    "* [CondaInstall](https://distributed.dask.org/en/stable/plugins.html#distributed.diagnostics.plugin.CondaInstall).\n",
```

Reply:
Same for me. @Retribution98, please take a look

Reply:
Yes, thanks for your suggestion!